CondTools/Geometry/test/me0geometrywriter.py

```diff
@@ -14,6 +24,9 @@
 
 process.ME0GeometryWriter = cms.EDAnalyzer("ME0RecoIdealDBLoader")
 
+#process.ME0GeometryWriter = cms.EDAnalyzer("ME0RecoIdealDBLoader",
+#                                           fromDD4Hep = cms.untracked.bool(False))
```

Comment:
Shouldn't these lines be uncommented now, and line 25 deleted? The old DD way should work with `fromDD4hep` set to false, right?

Reply:
@cvuosalo The old DD way is the default (i.e fromDD4Hep_ = iC.getUntrackedParameter<bool>("fromDD4Hep", false); inside the ME0RecoIdealDBLoader Class) ad I did for GEM. I deleted the comments inside me0geometrywriter.py